server/src/test/java/io/crate/analyze/SnapshotRestoreAnalyzerTest.java

```diff
@@ -336,14 +328,6 @@ public void testRestoreSinglePartitionToUnknownTable() throws Exception {
         assertThat(table.tableIdent()).isEqualTo(new RelationName(Schemas.DOC_SCHEMA_NAME, "unknown_parted"));
     }
 
-    @Test
-    public void testRestoreSingleExistingPartition() throws Exception {
-        assertThatThrownBy(
-            () -> analyze(e, "RESTORE SNAPSHOT my_repo.my_snapshot TABLE parted PARTITION (date=1395961200000)"))
-            .isExactlyInstanceOf(PartitionAlreadyExistsException.class)
-            .hasMessage("Partition '.partitioned.parted.04732cpp6ksjcc9i60o30c1g' already exists");
```

Comment:
I removed check because actually it's possible to restore into existing table, one should just close it.

There is even TODO to allow it for open indices
https://github.com/crate/crate/blob/master/server/src/main/java/org/elasticsearch/snapshots/RestoreService.java#L562-L565

I think once feature with renaming is implemented we can have a RestoreServiceTest confirming that instead of "already exists" users get a better hint like "close, drop or rename table"